buildSrc/src/main/kotlin/org/cqfn/save/buildutils/SpringBootConfiguration.kt

```diff
@@ -82,7 +82,7 @@ fun Project.configureSpringBoot(withSpringDataJpa: Boolean = false) {
 
     tasks.withType<BootRun>().configureEach {
         if (Os.isFamily(Os.FAMILY_MAC)) {
-            environment["SPRING_PROFILES_ACTIVE"] = "${environment["SPRING_PROFILES_ACTIVE"] ?: "dev"},mac"
+            environment["SPRING_PROFILES_ACTIVE"] = "dev,mac"
```

Comment:
What about `secure` profile for backend?

Reply:
Will add it now